### PR TITLE
new debug menu

### PR DESCRIPTION
**Pull request by HailSanta**

### CODE REVIEW

File: src/dx/config.h [hunk 1]
```diff
@@ -5,7 +5,7 @@
 #define DX_DEBUG_MENU 1
 
 /// Loads last used save file
-#define DX_QUICK_LAUNCH 0
+#define DX_QUICK_LAUNCH 1
```
Comment (nanaian) [inline src/dx/config.h:8]: Can you remove this diff

**Review by nanaian — COMMENTED** [review]
incredibly impressive

File: src/dx/config.h [hunk 2]
```diff
@@ -1,6 +1,9 @@
 #ifndef DX
 #define DX
 
+/// Set the version string that appears on the file select menu.
+#define DX_MOD_VERSION_STRING "??? ?.?.?"
```
Comment (nanaian) [inline src/dx/config.h:5]: Turn this off by default, and give a sensible example. Something like
```suggestion
/// Version string that appears on the file select menu.
//#define DX_MOD_VERSION_STRING "Awesome romhack 0.1.0"
```

Comment (HailSanta) [reply src/dx/config.h]: Added a default name "New Mod" and an explanation that you can comment it out to disable it. I think modders should be encouraged to display this information by default, though obviously not be required to do so.

**Pull request merged**